elastic/kibana
# fix(slo): Add slo-settings saved object to the slo feature registration

## Summary

This PR register the slo settings saved object with the slo feature.

**How to test**
1. Create some SLOs with an admin account
1. Setup a new role with a **read** access to the Observability > SLO feature, and the `*` index -> `read`
2. Setup a new user with that role
3. Go to the SLO page
4. Notice /api/observability/slo/settings and /api/observability/slos api are failing with Forbidden: `Unable to get slo-settings`
5. Checkout this branch
6. Errors should be gone.
